### PR TITLE
[zh] sync install kubectl

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh/docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -60,44 +60,97 @@ Using the latest version of kubectl helps avoid unforeseen issues.
...
+   使用校验和文件检查 kubectl 可执行二进制文件：
+
+   ```bash
+   echo "$(<kubectl.sha256) kubectl" | sha256sum --check
    ```
 
+   <!--
+   If valid, the output is:
+   -->
+   如何合法，则输出为：
```
Comment (tanjunchen) [inline content/zh/docs/tasks/tools/install-kubectl.md:107]: 如何合法?

File: content/zh/docs/tasks/tools/install-kubectl.md [hunk 2]
```diff
@@ -60,44 +60,97 @@ Using the latest version of kubectl helps avoid unforeseen issues.
...
+
+   ```bash
+   kubectl: OK
+   ```
+
+   <!--
+   If the check fails, `sha256` exits with nonzero status and prints output similar to:
+   -->
+   如何检查失败，则 `sha256` 退出且状态值非 0 并打印类似如下输出：
```
Comment (tanjunchen) [inline content/zh/docs/tasks/tools/install-kubectl.md:116]: 如何?

File: content/zh/docs/tasks/tools/install-kubectl.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/tasks/tools/install-kubectl.md:107]: good catch

File: content/zh/docs/tasks/tools/install-kubectl.md (hunk 2 above)
Comment (tengqm) [reply content/zh/docs/tasks/tools/install-kubectl.md:116]: good catch

**Review by ydFu — APPROVED** [review]

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**